app/main.py

```diff
@@ -1,2 +1,101 @@
+from dataclasses import dataclass
+from typing import Hashable, Any, Iterator
+
+INITIAL_CAPACITY = 8
```

Comment:
these should be class variables inside of Dictionary since it is part of it implementation